ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-release-4.10.yaml

```diff
@@ -0,0 +1,83 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
+binary_build_commands: make install
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: kubevirt-must-gather
+releases:
+  initial:
+    integration:
+      name: "4.10"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.10"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: |-
+  (
+    cd tests
+    go test -c -o must-gather.test .
+  )
+  mv tests/must-gather.test /go/bin/
+tests:
+- as: kubevirt-must-gather-e2e-azure
+  steps:
+    cluster_profile: azure4
+    test:
+    - as: deploy
+      cli: latest
+      commands: |
+        ./automation/deploy_hco.sh
+        ./automation/create_workloads.sh
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    - as: test-default
+      cli: latest
+      commands: |
+        oc adm must-gather --image="${KMG_IMAGE}" --dest-dir=${ARTIFACT_DIR}/must-gather-output
+        (cd ${ARTIFACT_DIR}; KUBECONFIG=${KUBECONFIG} ACK_GINKGO_DEPRECATIONS=1.16.4 /go/bin/must-gather.test --ginkgo.focus=level:product --junit-output=${ARTIFACT_DIR}/junit/report.xml --ginkgo.v)
```

Comment:
shouldn't it be 1.17.* also?
same in line 71

Reply:
> shouldn't it be 1.17.* also? same in line 71

Do you mean `ACK_GINKGO_DEPRECATIONS`? this is not related to the golang version. It related to ginkgo moving to version 2.0. this env var prevents the ginkgo warnings.